modules/serverless-rn-1-4-0.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-4-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.4.0
+
+[id="new-features-1-4-0_{context}"]
+== New features
+* {ServerlessProductName} 1.4.0 is available on {product-title} 4.2 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.11.0.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.11.0.
+* Knative Client is now available for download through the *Command Line Tools* page in the {product-title} web console.
+
+[id="fixed-issues-1-4-0_{context}"]
+== Fixed issues
```

Comment:
We could add here:

```
- Fixed a bug where Knative Services could not be reached from outside of the installed ServiceMesh on their cluster-local address
```

https://issues.redhat.com/browse/SRVKS-378 for reference of that bug.